db/migrate/20161219222535_add_encrypted_phone.rb

```diff
@@ -0,0 +1,6 @@
+class AddEncryptedPhone < ActiveRecord::Migration
+  def change
+    add_column :users, :encrypted_phone, :text, default: ''
```

Comment:
I should actually remove the default. `phone` and `email` differ in that the `email` can never be `nil` while the `phone` can. That's a Devise thing I had to debug my way through.

You can see the difference in the `def email` and `def phone` methods in `UserEncryptedAttributeOverrides`.